src/core/linter-rules/check-punctuation.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Linter rule "check-punctuation". Makes sure the there are no punctuations missing at the end of a <p>
+ *   in the ReSpec config.
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "check-punctuation";
+
+const meta = {
+  en: {
+    description: "`<p>` tags should end with either of . | : | ! | ? ",
```

Comment:
Apart from us 🤓s, it's a bit uncommon to use "|" to separate things. Let's use "," instead.  

```JS
description: "Found `<p>` element(s) without an ending punctuation mark.",
howToFix: `Add one of the following add one of the following punctuation marks to the offending element(s): ${punctuationMarks.join(", ")}`,
```